src/extractors/custom/news.mynavi.jp/index.js

```diff
@@ -0,0 +1,47 @@
+export const NewsMynaviJpExtractor = {
+  domain: 'news.mynavi.jp',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
+  },
+
+  author: {
+    selectors: [
+      'body > div.wrapper > div.container > div.body > main > div.article-author > a',
```

Comment:
If possible, I'd try to make this a little more generic. Extremely specific selectors like this can be brittle if the html ever changes. If, for example, `main div.article-author` worked, I'd suggest going with that.

Reply:
I fixed as below.
```
      'main div.article-author a.article-author__name',
```

Reply:
I fixed as below.
```
      'main article div',
```